sweddit_combined/decisions/ChineseFormables.txt

```diff
@@ -1,143 +1,76 @@
-			owns_core_province = 690
-			owns_core_province = 2139
-		}
-		effect = {
-			1816 = {
-				add_base_tax = 2
-				add_base_production = 2
-				add_base_manpower = 2
-				move_capital_effect = yes
-			}
-			change_tag = RCH
-			remove_non_electors_emperors_from_empire_effect = yes
-			china_superregion = {
-				limit = {
-					NOT = { owned_by = ROOT }
-				}
-				add_permanent_claim = RCH
-			}
-			add_prestige = 20
-			set_government_rank = 2
-			if = {
-				limit = { has_custom_ideas = no }
-				country_event = { id = ideagroups.1 } #Swap Ideas
-			}
-			country_event = { id = warringchina.2 } #Swap Government
-			set_country_flag = formed_beiyang_china
-		}
-		ai_will_do = {
-			factor = 1
-		}
-		ai_importance = 400
-	}
-	
 	form_chinese_empire = {
 		major = yes
 		potential = {
 			NOT = { has_country_flag = formed_chinese_empire }
 			normal_or_historical_nations = yes
-			was_never_end_game_tag_trigger = yes
+			OR = {
+				tag = MNG
+				was_never_end_game_tag_trigger = yes
+			}
 			culture_group = east_asian
 			is_colonial_nation = no
-			NOT = { tag = RCH }
 		}
 		provinces_to_highlight = {
 			OR = {
-				region = south_china_region
-				region = north_china_region
+				province_id = 1816 #Beijing
+				province_id = 691 # Jinan
+				province_id = 693 #Taiyuan
+				province_id = 688 #Kaifeng
+				province_id = 685 #Yangzhou
+				province_id = 1821 #Nanjing
+				province_id = 684 #Hangzhou
+				province_id = 667 #Canton
+				province_id = 2157 #Waichow
+				province_id = 1829 #Quangzhou
+				province_id = 669 #Fuzhou
+				province_id = 2174 #Hengyang
+				province_id = 662 #Yunnan
+				province_id = 682 #Wuchang
+				province_id = 700 #Xi'an
+				province_id = 680 #Chongqing
+				province_id = 679 #Chengdu
+				province_id = 699 #Lanzhou
 			}
 			OR = {
 				NOT = { owned_by = ROOT }
 				NOT = { is_core = ROOT }
 			}
 		}
 		allow = {
-			adm_tech = 15
+			adm_tech = 10
 			stability = 3
 			is_free_or_tributary_trigger = yes
 			culture_group = east_asian
 			is_at_war = no
 			AND = {
-				owns_core_province = 1821 #Nanjing
 				owns_core_province = 1816 #Beijing
+				owns_core_province = 691 # Jinan
+				owns_core_province = 693 #Taiyuan
+				owns_core_province = 688 #Kaifeng
+				owns_core_province = 685 #Yangzhou
+				owns_core_province = 1821 #Nanjing
+				owns_core_province = 684 #Hangzhou
+				owns_core_province = 667 #Canton
+				owns_core_province = 2157 #Waichow
+				owns_core_province = 1829 #Quangzhou
+				owns_core_province = 669 #Fuzhou
 				owns_core_province = 2174 #Hengyang
+				owns_core_province = 662 #Yunnan
+				owns_core_province = 682 #Wuchang
 				owns_core_province = 700 #Xi'an
-				owns_core_province = 667 #Beijing
+				owns_core_province = 680 #Chongqing
+				owns_core_province = 679 #Chengdu
+				owns_core_province = 699 #Lanzhou
 			}
-			OR = {
-				south_china_region= {
-					type = all
-					owned_by = ROOT
-				}
-				north_china_region= {
-					type = all
+			calc_true_if = {
```

Comment:
> One of the following must be true
>   All provinces in the South China region:
>      Owned by X
>   All provinces in the North China region:
>      Owned by X

Why is this removed and exchanged for what is here now?

Reply:
Mitt egna förslag för att undvika probelemet med att undvika problemet med Taiwan när man vill forma Kina. Spelmässigt ungefär lika svårt men lite mer dynamiskt. Kan ta bort.